TUT/STA130F24_TUT06_Oct25.ipynb

```diff
@@ -0,0 +1,12137 @@
+       "\n",
+       "// Listen for the clearing of the current output cell\n",
+       "var outputEl = gd.closest('.output');\n",
+       "if (outputEl) {{\n",
+       "    x.observe(outputEl, {childList: true});\n",
+       "}}\n",
+       "\n",
+       "                        })                };                });            </script>        </div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "broadway['log_avg_ticket_price'] = np.log(broadway['avg_ticket_price'])\n",
+    "broadway['weekly_gross_overall_in_100k'] = broadway['weekly_gross_overall']/100000 \n",
+    "fig = px.scatter(broadway, x='weekly_gross_overall_in_100k', \n",
+    "                 y='log_avg_ticket_price', trendline='ols')\n",
+    "fig.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e99b40a9",
+   "metadata": {},
+   "source": [
+    "## Feedback + Discussion of Assumptions [10 minutes]\n",
+    "\n",
+    "### (Use images for reference) \n",
+    "> ![](https://stats.libretexts.org/@api/deki/files/1599/imageedit_3_7796089347.png?revision=1)\n",
+    "<img src=\"https://sphweb.bumc.bu.edu/otlt/MPH-Modules/BS/R/R5_Correlation-Regression/Illustration%20of%20Simple%20Linear%20Regression.gif\" alt=\"\" style=\"width: 450px;\"/>\n",
+    "\n",
+    "\n",
+    "**The error terms follow the normal distribution, as assumption which can be verified by inspecting that the empirical distribution of the residuals appears to be bell shaped.**\n",
+    "\n",
+    "**The assumption of homoscedasticity is satsified iff the variance of the error terms is constant, or does not depend on the independent variable. (important distinction in graphs above, in the first graph, we observe what is known as _\"hetroscedasticity\"_)**\n",
+    "\n",
+    "**The linear form states that the change in the dependent variable must be proportional to the change in independent variable (consider the equation of simple linear regression).**\n",
+    "\n",
+    "**There is no relationship between $Y_i$ (the outcome variables) execpt for how they depend on $x_i$ (the independent variables) (known as the assumption of independence).**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "43185ef7",
+   "metadata": {},
+   "source": [
+    "## Demo [15 minutes]\n",
+    "\n",
+    "- Use ChatGPT to create a dataset having several variables related to a musical show\n",
+    "- Identify two sets of predictor (independent) and outcome (variables) with students and plot them.\n",
+    "(one of the independent variables should be a indicator variable which should be used to transition into next part of tutorial\n",
+    "- Demonstrate using smf how a trendline can be plotted and how we can use it to find estimates of the parameters of the linear regression model.\n",
+    "- Demonstrate how to display the summary of the smf model and highlight key areas **(Coefficients of Intercept and Slope, R-squared value, p value)**"
```

Comment:
Good, I'm glad you're using `smf` -- that's what we should be using here.

And, this appears to potentially be the example that I'm looking for here, with the following caveats.

1. indicator variables go into the Nov04 LEC
2. you should not be doing any MLR... that's the next week... not sure if you're doing that or not yet... haven't read far enough; but, I'm going to get you connected with Serena to make sure this transition is ironed out smoothy, in case that's an issue.  It does not appear there as been any connection between you and Serena so far, and I believe I've (a) indicated that you two should coordinate the transition of your materials together, and (b) that Serena should review your work prior to it arriving to me. Both of these I think will probably still be appropriate given that you will have a major iteration on this material in front of you based on my comments.